src/styles/globals.css

```
@@ -66,24 +66,25 @@
   --sidebar-primary-foreground: oklch(1 0 0);
   --sidebar-accent: oklch(0.205 0 0);
   --sidebar-accent-foreground: oklch(1 0 0);
   --sidebar-border: oklch(1 0 0 / 8%);
-  --sidebar-ring: oklch(0.546 0.245 262.881);
+  --sidebar-ring: oklch(0.65 0.28 305);
 }
 
 @theme inline {
-  --font-mono: var(--font-mono);
+  --font-sans: var(--font-inter), ui-sans-serif, system-ui, sans-serif;
+  --font-mono: var(--font-jetbrains), ui-monospace, SFMono-Regular, monospace;
```

Stylelint (17.10.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)